Modules/Scripted/Home/HomeLib/segmentation_model.py

```diff
@@ -16,24 +16,28 @@
 
 
 class SegmentationModel:
-  class ModelSource(enum.Enum):
-      USE_PTH = 0
-      USE_LOCAL_ZIP = 1
-      USE_DOCKER_ZIP = 2
+  class NoValue(enum.Enum):
+    def __repr__(self):
+      return '<%s.%s>' % (self.__class__.__name__, self.name)
+
+  class ModelSource(NoValue):
+      TORCHSCRIPT_ONLY = 'TorchScript only'
```

Comment:
The pth file that was saved via `torch.save` is not TorchScript, is it? I think you have to save with `torch.jit.script` and `torch.script.save` to get it as TorchScript.

The "TorchScript only" terminology may then be somewhat misleading as well.

Reply:
Looks great, thank you!